### DISCUSSION TITLE
방장 부방장 체크

### DISCUSSION THREAD

**Discussion by veplay** [storycraft/node-kakao#634] [Q&A]
방장 부방장이 체크가 가능할까요?

v3 이용중입니다

userInfo.MemberType 로 받아와도 안되구.. 어떻게 해야할지 고수분들 부탁드립니다 ㅠㅠ

**Comment by jipsoft1** [comment]
getMemberType(user: ChatUser): OpenMemberType;
    getMemberTypeId(userId: Long): OpenMemberType;

오픈채팅의 방장/부방장/일반유저 확인은 위 함수로 확인이 가능합니다.
OpenChatChannel 객체의 멤버함수입니다. ChannelManager를 통해 채널을 가져오든가 해서 사용하시면 됩니다.

**Reply by veplay** [reply]
혹시 chat.Channel.getMemberType(chat.Sender) 맞을까요?

값이 인원이 적은방에서는 잘넘어오는데,

1000명이 넘은방에서는 제대로 안들어오고 전부 undefined 으로 들어오네요..
